src/pages/FlowRun/Run-Config-Tile.vue

```diff
@@ -0,0 +1,148 @@
+<script>
+import { runConfigs } from '@/utils/runConfigs'
+import Label from '@/components/Label'
+export default {
+  components: {
+    Label
+  },
+  props: {
+    flowRun: {
+      required: true,
+      type: Object
+    }
+  },
+  data() {
+    return {}
+  },
+  computed: {
+    runConfig() {
+      return runConfigs[this.flowRun.run_config.type]
+    },
+    argList() {
+      let argList = {}
+      for (let arg in this.flowRun.run_config) {
+        const info = this.runConfig.args.find(a => a.arg === arg)
+        if (info?.label) {
+          argList[info.label] = this.flowRun.run_config[arg]
+        } else if (
+          arg.includes('job_template') ||
+          arg.includes('task_definition')
+        ) {
+          const override = this.runConfig.args[0].options.find(
+            a => a.arg === arg
+          )
+          argList[override.label] = this.flowRun.run_config[arg]
+        } else if (arg === 'labels' && this.flowRun.labels.length) {
+          argList['Labels'] = this.flowRun.labels
+        }
+      }
+      return argList
+    }
+  },
+  methods: {}
+}
+</script>
+
+<template>
+  <v-card class="px-8 full-height">
+    <div class="text-h5 py-8">Custom Run Configurations</div>
+    <div
+      v-if="flowRun.run_config == null"
+      class="text-center position-absolute center-absolute text-h5 utilGrayDark--text"
+      style="z-index: 1;"
+    >
+      This run has no
+      <span class="font-weight-medium">
+        <v-icon small color="primary" class="ml-1">
+          fa-cogs
+        </v-icon>
+        Run Config</span
+      >! You can learn how to set a run config in the
+      <a
+        href="https://docs.prefect.io/orchestration/ui/flow.html#run"
+        target="_blank"
+      >
+        Flow Run Docs</a
+      >.
+    </div>
+    <span v-else>
+      <div class="row-divider pt-2 pb-8">
+        <v-row>
+          <v-col cols="4">
+            <v-card-subtitle class="text-h6">Agent</v-card-subtitle>
+          </v-col>
+          <v-col cols="8">
+            <div class="d-flex align-center agent-type">
+              <div class="text-center" style="width: 50px;">
+                <i :class="runConfig.icon" class="fa-2x pi-2x"> </i>
+              </div>
+              <div>
+                <v-card-text class="config-value">{{
+                  runConfig.label
+                }}</v-card-text>
+              </div>
+            </div>
+          </v-col>
+        </v-row>
+      </div>
+      <div
+        v-for="(arg, i) of Object.keys(argList)"
+        :key="i"
+        class="row-divider pt-2 pb-8"
+      >
+        <v-row>
+          <v-col cols="4">
+            <v-card-subtitle class="text-h6">{{ arg }}</v-card-subtitle>
+          </v-col>
+          <v-col cols="8">
+            <v-card-text v-if="arg === 'Labels'">
+              <span v-for="(label, j) in argList[arg]" :key="j"
+                ><Label disabled>{{ label }}</Label></span
```

Comment:
I think we need to add a tiny bit of padding between labels: 

<img width="765" alt="Screen Shot 2021-07-14 at 3 43 55 PM" src="https://user-images.githubusercontent.com/40272060/125641995-5209ef68-edb9-4076-ac0a-86450f483b62.png">